Zend/zend_attributes_arginfo.h

```diff
@@ -71,3 +111,31 @@ static zend_class_entry *register_class_AllowDynamicProperties(void)
 
 	return class_entry;
 }
+
+static zend_class_entry *register_class_SensitiveParameter(void)
+{
+	zend_class_entry ce, *class_entry;
+
+	INIT_CLASS_ENTRY(ce, "SensitiveParameter", class_SensitiveParameter_methods);
+	class_entry = zend_register_internal_class_ex(&ce, NULL);
+	class_entry->ce_flags |= ZEND_ACC_FINAL;
+
+	return class_entry;
+}
+
+static zend_class_entry *register_class_SensitiveParameterValue(void)
+{
+	zend_class_entry ce, *class_entry;
+
+	INIT_CLASS_ENTRY(ce, "SensitiveParameterValue", class_SensitiveParameterValue_methods);
+	class_entry = zend_register_internal_class_ex(&ce, NULL);
+	class_entry->ce_flags |= ZEND_ACC_FINAL;
```

Comment:
Adding `|ZEND_ACC_FORBID_DYNAMIC_PROPERTIES` seems like it'd make sense, dynamic property support ($sensitiveParameter->dynamicProp = 123;) is going away in 9.0

Reply:
Makes sense, done.